src/pint/utils.py

```diff
@@ -1667,3 +1667,17 @@ def remove_dummy_distance(c):
             "Do not know coordinate frame for %r: returning coordinates unchanged" % c
         )
         return c
+
+
+def info_string(prefix_string="# ", comment=None):
+    import pint
+    import getpass
+    import platform
+    import datetime
+
+    s = f"Created: {datetime.datetime.now().isoformat()}\nPINT_version: {pint.__version__}\nUser: {getpass.getuser()}\nHost: {platform.node()}\nOS: {platform.platform()}"
```

Comment:
Can we do better than `getpass`? Unfortunately the containers used by NANOGrav and others often mean that `getpass.getuser()` will often return something unhelpful (`jovyan` for all users on the NANOGrav notebook server). Could one drop in the git user information if that was available?

Reply:
I tried to find something that was cross-platform.  Maybe this isn't the answer.  I tried to get the git info using gitpython, but I'm not sure how it will work if the global git config isn't available